README.md

```diff
@@ -74,6 +74,11 @@ Once you have created your TwiML app, [configure your Twilio phone number](https
 
 4. Run the application.
 
+   Make sure MongoDB is running before starting the application. See how to start the MongoDB service on
+   [Windows](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/#start-mongodb-community-edition-as-a-windows-service), [MacOS](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/#run-mongodb-community-edition), or [Linux](https://docs.mongodb.com/manual/administration/install-on-linux/) (choose your Linux distribution and then see "Run MongoDB Community Edition" in the installation instructions).
```

Comment:
Since this is necessary before running the app, would it be clearer to make this its own numbered step? Just an idea!

Reply:
Makes sense! I'll split it out.